### PR TITLE
Fetch prebuilt debezium-server from s3

### PR DESCRIPTION
**Pull request by rahulb-yb**

### CODE REVIEW

File: installer_scripts/install-yb-voyager-debezium [hunk 1]
```diff
@@ -8,8 +8,9 @@ trap on_exit EXIT
 LOG_FILE=/tmp/install-yb-voyager-debezium.log
 YUM="sudo yum install -y -q"
 
+# Remember to update the S3 repo with the correct version of pre-biult debezium-server, if any change in these values.
 DEBEZIUM_VERSION="2.2.0-SNAPSHOT"
-DEBEZIUM_COMMIT_HASH="7164f6280183d7d7c92088537792e9200defad09"
+DEBEZIUM_COMMIT_HASH="4d513804b713505a4e5c8479d10c92f047f2a253"
```
Comment (sanyamsinghal) [inline installer_scripts/install-yb-voyager-debezium:13]: @rahulb-yb please update it with the latest commit hash
`4c892e5e3d5b4fad26daa4bf6e253cff0eb454a7`

File: .github/workflows/pg-migtests.yml [hunk 2]
```diff
@@ -43,6 +43,20 @@ jobs:
...
+      with:
+        args: s3 cp s3://voyager-automation-data/debezium-latest.tar.gz .
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.RAHULB_S3_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.RAHULB_S3_SECRET_ACCESS_KEY }}
+        AWS_DEFAULT_REGION: "us-west-2"
+    
+    - name: Unpack debezium-server
+      run: |
+        sudo tar -xvzf debezium-latest.tar.gz -C /opt/
```
Comment (amit-yb) [inline .github/workflows/pg-migtests.yml:57]: Recently the dbzm installer script is modified to install debezium-server at `/opt/yb-voyager/`.

**Review by amit-yb — APPROVED** [review]
LGTM with a minor suggestion.

File: installer_scripts/install-yb-voyager-debezium [hunk 3]
```diff
@@ -222,7 +223,7 @@ install_debezium_server() {
 	sudo tar -xvzf debezium-server/debezium-server-dist/target/${debezium_server_dist_filename} -C /opt/ 1>&2
 	# cleanup
 	cd ..
-	rm -r debezium-${DEBEZIUM_COMMIT_HASH}
+	#rm -r debezium-${DEBEZIUM_COMMIT_HASH}
```
Comment (sanyamsinghal) [inline installer_scripts/install-yb-voyager-debezium:226]: @rahulb-yb  why commenting this line?

**Pull request merged**